tests/validators/test_list.py

```diff
@@ -391,9 +396,9 @@ def f(v: int) -> int:
         {
             'type': 'too_long',
             'loc': (),
-            'msg': 'List should have at most 10 items after validation, not 11',
+            'msg': 'List should have at most 10 items after validation, not 15',
```

Comment:
Since the actual size of the list is 15 and not 11 I decided to change the values.